src/components/layout/card/Card.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types'
+import TopBar from '../topBar/TopBar'
+import './card.css'
+
+class Card extends Component {
+
+static propTypes ={
+    size: PropTypes.oneOf(['large', 'small']),
+    cardName: PropTypes.string.isRequired,
+    cardType: PropTypes.oneOf(['green', 'grey'])
```

Comment:
`cardType: PropTypes.oneOf(['green', 'grey'])`
green replace with eg. primary or success
grey replace with eg. neutral
you can also use `variant` keyword instead of `cardType`